cmd/fa-om-exporter/main.go

```diff
@@ -2,29 +2,58 @@ package main
 
 import (
 	"context"
-	"flag"
 	"fmt"
 	"log"
 	"net/http"
+	config "purestorage/fa-openmetrics-exporter/internal/config"
 	collectors "purestorage/fa-openmetrics-exporter/internal/openmetrics-exporter"
 	client "purestorage/fa-openmetrics-exporter/internal/rest-client"
 	"strings"
+	"os"
+	"io"
 
+	"gopkg.in/yaml.v3"
+	"github.com/akamensky/argparse"
 	"github.com/prometheus/client_golang/prometheus"
 	"github.com/prometheus/client_golang/prometheus/promhttp"
 )
 
-var version string = "1.0.3"
+var version string = "1.0.4"
 var debug bool = false
+var arraytokens config.FlashArrayList
 
 func main() {
 
-	host := flag.String("host", "0.0.0.0", "Address of the exporter")
-	port := flag.Int("port", 9490, "Port of the exporter")
-	d := flag.Bool("debug", false, "Debug")
-	flag.Parse()
-	addr := fmt.Sprintf("%s:%d", *host, *port)
+	parser := argparse.NewParser("pure-fa-om-exporter", "Pure Storage FA OpenMetrics exporter")
+	host := parser.String("a", "address", &argparse.Options{Required: false, Help: "IP address for this exporter to bind to", Default: "0.0.0.0"})
+	port := parser.Int("p", "port", &argparse.Options{Required: false, Help: "Port for this exporter to listen", Default: 9491})
```

Comment:
default port should be 9490. the FB OME is using this port. since this is a minor release, we should not consider a port change.